### ISSUE TITLE
Razor Class Library with Asp.Net 3.0

### ISSUE THREAD

**Issue by csutorasr** [aspnet/AspNetCore#8070]
### Describe the bug
I found that if I would like to build a Razor Class Library I cannot do that as I can't target `netstandard2.1` because of the missing SDK. The preview versions of `Microsoft.AspNetCore.Mvc` only supports `netcoreapp3.0`.

My page is not showing if using `netcoreapp3.0` as shown here: https://github.com/aspnet/AspNetCore/issues/4332

### To Reproduce
Create a RCL in VS2019

### Expected behavior

I would like to make a library that works with .NET Standard so I could use it anywhere. Should it be done with targeting `netcoreapp3.0`? Then docs are missing for RCL in 3.0.

### Additional context
I am using the preview2 of .NET Core 3.0

Maybe https://github.com/dotnet/standard/issues/1090 is related.

**Comment by NTaylorMullen** [comment]
Depending on the feature set you're looking for you'll want to target one of the various forms of .net standard. For instance, for our pre-3.0 work a lot of Asp.Net Core was targeting netstandard2.0.

If you find yourself in a position where it says missing SDK xyz chances are you haven't yet downloaded/installed the SDK corresponding to what you're trying to build (see below). In your case, you also ran into one of our big in progress issues with 3.0 (razor class libraries don't work), therefore hopefully you don't require 3.0 preview features. Ideally you shouldn't since that whole pipeline is still in preview.

For places to download sdks: https://dot.net
A great article explaining .net standard: https://docs.microsoft.com/en-us/dotnet/standard/net-standard

Hope this helps!

**Comment by aodl** [comment]
@NTaylorMullen is there an open issue for the problem(s) with razor class libraries in 3.0? (you mentioned above that razor class libraries don't work). I can't seem to find an open issue to follow though.

------------------------------

I've just started looking to play around moving from core 2.2 to 3.0, and it seems that RCLs work when you first build you're app after adding a reference to them. But then if you make any kind of modification that requires the referencing app to be rebuilt, the RCLs stop working (their content is no longer reachable). This work-once-but-not-on-subsequent-builds behaviour reminds me of a [bug fix](https://github.com/aodl/AspNetCore-Tooling/commit/c38391ca501e63f8cc06f63291f1383a335f9ed5) I issued a while ago regarding static files in RCLs (only being copied to output directory on first build, but not subsequent ones).
